src/main/java/com/agiletec/plugins/jacms/aps/system/services/resource/model/ImageResource.java

```diff
@@ -158,6 +159,14 @@ public void saveResourceInstances(ResourceDataBean bean, List<String> ignoreMeta
         logger.debug("Get image Metadata in Resource Action");
         Map<String, String> meta = new HashMap<>();
         try {
+
+            String extension = FilenameUtils.getExtension(file.getName());
```

Comment:
I wonder if it's better to rely on mime type instead of extension, wdyt?

Reply:
Done!